website/scripts/tables.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
```

Comment:
Here as an example, should be generated using https://github.com/cloudquery/plugin-sdk/pull/347